pkgs/applications/editors/vim/plugins/overrides.nix

```
@@ -134,12 +126,12 @@
   # search-and-replace.nvim dependencies
   fd,
   sad,
-  # ethersync vim plugin
-  ethersync,
 }:
 self: super:
 let
   luaPackages = neovim-unwrapped.lua.pkgs;
+  buildVimPlugin = throw "New plugin definitions should be done outside `overrides.nix`";
+  buildNeoVimPlugin = throw "New plugin definitions should be done outside `overrides.nix`";
```

Lets add a clarifying comment so that no one thinks it is safe to "clean up" these unused variables.

Not sure about my wording, but here's one possible suggestion:

```suggestion
  # Ensure the vim plugin builders are not used in this file.
  # If they are used, these stubs will throw.
  buildVimPlugin = throw "New plugin definitions should be done outside `overrides.nix`";
  buildNeoVimPlugin = throw "New plugin definitions should be done outside `overrides.nix`";
```